src/Runner/CliTester.php

```diff
@@ -228,9 +230,6 @@ private function createRunner()
 	/** @return string */
 	private function prepareCodeCoverage()
 	{
-		if (!$this->interpreter->hasXdebug()) {
-			throw new \Exception("Code coverage functionality requires Xdebug extension (used {$this->interpreter->getCommandLine()})");
-		}
```

Comment:
Imho, this should stay there with modified message. And as a workaround, `ZendPhpDbgInterpreter::hasXDebug()` should return TRUE.